### ISSUE TITLE
Switch from nabble to gitter

### ISSUE THREAD

**Issue by nicolas-f** [Universite-Gustave-Eiffel/NoisePlanet#86]
It's a debate about whether we should switch to gitter (https://gitter.im/NoiseModelling/community) instead of the mailing list (http://orbisgis.3871844.n2.nabble.com) on the noise planet webpage (https://noise-planet.org/noisemodelling.html#community)

@ebocher @Picaut @pierromond @gpetit 

The issue is we have only emails and our answers are not shared with the community.

**Comment by gpetit** [comment]
Yes we should find a way to optimize this point (since Nabble and more widely mailing lists are quite "old school" and have sometimes security / spam issues).

IMO, we have to distinguish the questions and the discussions:
- questions / remarks : they should be public so thath answers can benefit to the all community. Nabble was a solution ... but maybe we should just focus on Github issues?
- discussions : they may be public or private, bilateral or widely opened. In this case, Gitter may be a good option (other community are using IRC channels. If I understand well, this is more or less the same approach).

**Comment by pierromond** [comment]
I like github issues + gitter solution.

**Comment by nicolas-f** [comment]
Also on the Contact page https://noise-planet.org/contact.html

How about adding a "Community" insert above the email link to limit email interactions to things that are private ?

**Comment by gpetit** [comment]
Yes we can add a message to clarify this aspect : only use contact email for private questions.
I will add this point in the list of updates for the website
